app/scripts/main.tsx

```diff
@@ -3,7 +3,7 @@ import { createRoot } from 'react-dom/client';
 import T from 'prop-types';
 import { BrowserRouter, Route, Routes, useLocation } from 'react-router-dom';
 
-import '@trussworks/react-uswds/lib/index.css';
+// import '@trussworks/react-uswds/lib/index.css';
```

Comment:
I think I read frmo uswds-react docs that we need to include this, but I don't really see the point of including this. @dzole0311 do you have more context?

Reply:
I was in-between commenting & uncommenting it myself in my PRs. Let's keep it commented or we remove it until we see a need for it.